src/pages/task/taskTabs/buildBaronAndAnnotations/AnnotationTicketsTable/AnnotationTicketsTable.tsx

```diff
@@ -0,0 +1,243 @@
+                icon={null}
+                placement="topRight"
+                title={`Do you want to delete this ${issueString}?`}
+                onConfirm={() => {
+                  handleRemove(url, issueKey);
+                }}
+                okText="Yes"
+                cancelText="Cancel"
+              >
+                {children}
+              </Popconfirm>
+            ),
+            altWrapper: (children: JSX.Element) => (
+              <Tooltip trigger={children}>
+                You are not authorized to edit failure details
+              </Tooltip>
+            ),
+            children: (
+              <Button
+                size="small"
+                data-cy={`${issueKey}-delete-btn`}
+                leftGlyph={<Icon glyph="Trash" />}
+                disabled={!userCanModify}
+              />
+            ),
+          })}
+        </ButtonContainer>
+      ),
+    },
+  ];
+
+  const [removeAnnotation] = useMutation<
+    RemoveAnnotationIssueMutation,
+    RemoveAnnotationIssueMutationVariables
+  >(REMOVE_ANNOTATION, {
+    onCompleted: () => {
+      dispatchToast.success(`Successfully removed ${issueString}`);
+    },
+    onError(error) {
+      dispatchToast.error(
+        `There was an error removing the ${issueString}: ${error.message}`
+      );
+    },
+    refetchQueries: ["GetSuspectedIssues", "GetIssues"],
+  });
+
+  const [moveAnnotation] = useMutation<
+    MoveAnnotationIssueMutation,
+    MoveAnnotationIssueMutationVariables
+  >(MOVE_ANNOTATION, {
+    onCompleted: () => {
+      dispatchToast.success(
+        `Successfully moved ${issueString} to ${
+          isIssue ? "suspected issues" : "issues"
+        }`
+      );
+    },
+    onError(error) {
+      dispatchToast.error(
+        `There was an error moving the ${issueString}: ${error.message}`
+      );
+    },
+    refetchQueries: ["GetSuspectedIssues", "GetIssues"],
+  });
+
+  const handleRemove = (url: string, issueKey: string): void => {
+    const apiIssue = {
+      url,
+      issueKey,
+    };
+    removeAnnotation({ variables: { taskId, execution, apiIssue, isIssue } });
+    const analyticsType = isIssue
+      ? "Remove Annotation Issue"
+      : "Remove Annotation Suspected Issue";
+    annotationAnalytics.sendEvent({
+      name: analyticsType,
+    });
+  };
+
+  const handleMove = (apiIssue: {
+    url: string;
+    issueKey: string;
+    confidenceScore: number;
+  }): void => {
+    moveAnnotation({ variables: { taskId, execution, apiIssue, isIssue } });
+    const analyticsType = isIssue
+      ? "Move Annotation Issue"
+      : "Move Annotation Suspected Issue";
+    setSelectedRowKey(apiIssue.issueKey);
+    annotationAnalytics.sendEvent({
+      name: analyticsType,
+    });
+  };
+
+  // SCROLL TO added Issue
+  // Will add a span with a ref to the row that matches the selectedRowKey
+  // And will scroll to that ref.
+  const rowRef = useRef<HTMLSpanElement>(null);
+
+  useEffect(() => {
```

Comment:
did you mean to make the dep array undefined instead of empty? an undefined dep will run the useEffect cb on every re-render while an empty array will only run it once